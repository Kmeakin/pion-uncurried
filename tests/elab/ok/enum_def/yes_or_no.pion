enum YesOrNo {
    Yes,
    No,
}

let y = Yes;
let n = No;