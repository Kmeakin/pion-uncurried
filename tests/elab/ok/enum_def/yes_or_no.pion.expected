enum YesOrNo: Type {
    Yes: YesOrNo,
    No: YesOrNo,
}

let y: YesOrNo = Yes;

let n: YesOrNo = No;
